data/World/Forest Temple MQ.json

```diff
@@ -158,7 +183,11 @@
         "region_name": "Forest Temple Before Boss",
         "dungeon": "Forest Temple",
         "locations": {
-            "Forest Temple MQ Basement Chest": "True"
+            "Forest Temple MQ Basement Chest": "True",
+            "Forest Temple MQ Basement Pot 1": "True",
+            "Forest Temple MQ Basement Pot 2": "True",
+            "Forest Temple MQ Basement Pot 3": "True",
+            "Forest Temple MQ Basement Pot 4": "True"
```

Comment:
The first time I tried, I couldn't get drops from these pots. Do they share flags with other pots earlier in the dungeon?

Reply:
So I'll start by saying that shared flags are definitely not "respected" in rupee/pot/crate shuffle. Many scenes are updated in order to avoid scenarios like you have described. You can see which locations have their flags updated in LocationList.py. They will have a value in their Address field which corresponds to the ROM location of the actor. These items in forest were just an oversight (Actually the eye switch room crates were patched so the only ones that are still shared in MQ Forest should be the basement pots and the well recovery hearts). This can easily be fixed.

The main reason why the pots room flags are currently respected is that there were not enough temporary flags when the room was first added to account for all 44 pots. After creating the new drop flag system, I just never went back to update those because honestly I didn't feel like it :)

Reply:
Let me see if it's easy enough. I haven't hacked pots to drop collectibles with what I like to call "extended flags".  Not my top priority just yet. I want to get through these logic fixes.

Reply:
What's going on with the shared flags here?